source/includes/algo_development/_algorithmmanagement.md

```diff
@@ -699,63 +857,142 @@ algo.versions() #optional params: limit (<int>), marker (<string>), published (<
   # Use Python or the OpenAPI spec instead:
-  # https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  # https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```cli
   # This client does not currently support Algorithm Management.
   # Use Python or the OpenAPI spec instead:
-  # https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  # https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```r
   # This client does not currently support Algorithm Management.
   # Use Python or the OpenAPI spec instead:
-  # https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  # https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```ruby
   # This client does not currently support Algorithm Management.
   # Use Python or the OpenAPI spec instead:
-  # https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  # https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```java
   // This client does not currently support Algorithm Management.
   // Use Python or the OpenAPI spec instead:
-  // https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  // https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```scala
   // This client does not currently support Algorithm Management.
   // Use Python or the OpenAPI spec instead:
-  // https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  // https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```rust
   // This client does not currently support Algorithm Management.
   // Use Python or the OpenAPI spec instead:
-  // https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  // https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```nodejs
   // This client does not currently support Algorithm Management.
   // Use Python or the OpenAPI spec instead:
-  // https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  // https://algorithmia.com/developers/algorithm-development/algorithm-management
 ```
 
 ```php
 <?
   // This client does not currently support Algorithm Management.
   // Use Python or the OpenAPI spec instead:
-  // https://algorithmia.com/developers/algorithm-development/algorithm-management-api
+  // https://algorithmia.com/developers/algorithm-development/algorithm-management
 ?>
 ```
 
-To get info about every individual published version of your Algorithm, use `algo.versions()`
\ No newline at end of file
+To get info about every individual published version of your Algorithm, use `algo.versions()`
+
+## Get Algorithm Repository Host Connection Status
+
+`GET https://api.algorithmia.com/v1/algorithms/:owner/:algoname/scm/status`
+
```

Comment:
:bulb: Let's add a small explainer here: "If your algorithm's source code is hosted on a platform like GitHub, you can use this endpoint to determine if Algorithmia is still able to properly communicate with the repository host and fetch source code."

Reply:
Updated!